.pa11yci

```diff
@@ -0,0 +1,16 @@
+{
+  "defaults": {
+    "standard": "WCAG2AA",
+    "chromeLaunchConfig": {
+      "args": [
+        "--no-sandbox"
```

Comment:
@agoldstone93 please can you run a quick check to confirm that an explicit path will resolve the issue. See the `Grover` initialiser.

Add this to `chromeLaunchConfig`, `"executablePath": "/usr/bin/chromium-browser",`